pkg/process/net/network_linux.go

```diff
@@ -109,6 +113,7 @@ func ShouldLogTracerUtilError() bool {
 func newSystemProbe() *RemoteSysProbeUtil {
 	return &RemoteSysProbeUtil{
 		socketPath: globalSocketPath,
+		mu:         &sync.Mutex{},
```

Comment:
Since RemoteSysProbeUtil is itself a pointer, do we need the mutex to be a pointer as well?